### DISCUSSION TITLE
Location Moved

### DISCUSSION THREAD

**Discussion by AlexFinch** [KeithSloan/GDML#121] [General]
Just trying this out for the first time. Using FreeCAD 0.21.1 it seems that the location for storing a workbench has changed. I had to copy files  from  ~/.FreeCAD/Mod to ~/.local/share/FreeCAD/ in order for GDML workbench to appear in FreeCAD.

**Comment by KeithSloan** [comment]
It is best to install the workbench via Tools | Addon Manager, it will then install in the correct location.

From memory the location of workbenches changed after FreeCAD 0.19, were you using 0.19 prior to 0.21?

**Reply by AlexFinch** [reply]
Yes I first tried the version which came with my distribution but that was too old so I downloaded the latest version instead.
